app/models/item.py

```diff
@@ -0,0 +1,22 @@
+from flask_sqlalchemy import Model
+from flask import abort
+
+
+class Item(Model):
```

Comment:
Nice work using inheritance to add this useful behavior to the `Task` and `Goal` models.